packages/grid/_modules_/grid/components/cell/GridEmptyCell.tsx

```diff
@@ -22,6 +22,7 @@ export const GridEmptyCell = React.memo(function GridEmptyCell({
         lineHeight: `${height - 1}px`,
         minHeight: height,
         maxHeight: height,
+        padding: 0,
```

Comment:
This looks off topic

```suggestion
```

Reply:
The issue is that the cell always have a padding of 10px
But we do not want it on the GridEmptyCell.

Maybe we should add a class to this cell and set the padding to 0 in the rootStyle instead of here.

Reply:
yes